test/why.js

```diff
@@ -358,11 +358,14 @@ test('functions', function (t) {
 		var genFnStar = Function('return function* () {};')();
 		var genFnSpaceStar = Function('return function *() {};')();
 		var genNoSpaces = Function('return function*(){};')();
-		st.equal(
-			'second argument is a Generator; first is not',
-			isEqualWhy(fnNoSpace, genNoSpaces),
-			'generator and fn that are otherwise identical are not equal'
-		);
+		var reasonsMap = {
+      'second argument is a Generator; first is not': true,
```

Comment:
it's hard to put a `\t` into github's interface, so my example used spaces, but can you replace the spaces in this line with tabs?